_assets/sass/custom/_utilities.scss

```diff
@@ -24,3 +24,25 @@
   list-style-type: none;
   padding-left: 0px;
 }
+
+.page-type-info-box {
+  display: flex;
+
+  .icon-column {
+    img {
+      margin-top: 1px;
+      width: 22px;
+      max-width: 22px;
+    }
+  }
+
+  .content-column {
+    h3 {
+      display: inline;
+      font-size: 1em;
+      margin: 8px !important;
```

Comment:
I don't think you need !important here - when I remove it the style remains.

Reply:
Nice - I'm also no longer seeing this as necessary, which is great because `important` is bad news. Thanks for checking!